Source/MQTTnet/Client/MqttClient.cs

```diff
@@ -34,6 +34,9 @@ public class MqttClient : Disposable, IMqttClient
         private CancellationTokenSource _backgroundCancellationTokenSource;
         private Task _packetReceiverTask;
         private Task _keepAlivePacketsSenderTask;
+        private Task _publishPacketReceiverTask;
+
+        private AsyncQueue<MqttPublishPacket> _publishPacketReceiverQueue;
```

Comment:
I assume this must be disposed as soon as the client gets disposed.

Reply:
Done